### ISSUE TITLE
PDFs should be generated with accessibility features

### ISSUE THREAD

**Issue by Nealium104** [uklibraries/omeka-solr-shim#132]
**Is your feature request related to a problem? Please describe.**
PDFs that are generated by ExploreUK are not generated to have accessible features such as OCR'd text and tags.

**Describe the solution you'd like**
PDF's generated by ExploreUK should implement as many accessibility features for PDF generation as we can.

**Additional context**
Some introductory material:
https://www.portland.gov/officeofequity/documents/introduction-pdf-accessibility-0/download
https://www.section508.gov/create/pdfs/
https://www.ssa.gov/accessibility/developer_resources.html
https://www.w3.org/TR/WCAG20-TECHS/pdf
